### DISCUSSION TITLE
Cinnamon

### DISCUSSION THREAD

**Discussion by snoweuph** [Almamu/linux-wallpaperengine#92] [Q&A]
I wanted to ask if this is compatible with Cinnamon yet or does is still needs work to function cinnamon. btw I appreciate this project :+1:

**Comment by Almamu** [comment]
I have not tried it under cinnamon, so I cannot really say if it'll work or not. I'd guess it won't, but never hurts trying it first.

**Comment by RiverMoot** [comment]
Overall: **Yes**, BUT-

I haven't read through others' experiences so I don't know how much of this is Cinnamon-exclusive and how much is just the program in general.
If you have a particular wallpaper in mind, this is not for you, at least at this very moment. I managed to get it working, and I'm going through wallpapers, and overall here's what I've found:
- Wallpapers with fancy audio responsiveness don't work.
- Effects in wallpapers are VERY hit and miss.
- There's some weird stuff with resolution sometimes?
- Video-based wallpapers work best, but their framerate tanks when you actually put them on the desktop.
- A lot of wallpapers kinda just- don't work, but that might be on my part because I don't know if ALL the wallpapers are in the same directory.
- The wallpapers lag my computer a bit but it's very hard to tell.
- Having a wallpaper on more than one screen is TECHNICALLY possible, but it flickers between them and makes your computer lag lots.
- If you have more than one monitor, the other's wallpaper blanks into grey or black.
- Your wallpapers don't go back to normal after closing it (unless you restart.)

Personally I'm gonna wait until later in its development to use it. That being said, I think this looks rly cool and has a lot of potential! :O
